### PR TITLE
Add author

### PR DESCRIPTION
**Pull request by driusan**

### CODE REVIEW

**Comment by driusan** [comment]
Not seeing a green checkmark beside the commit makes me nervous.

File: paper.md [hunk 1]
```diff
@@ -29,9 +29,9 @@ authors:
 # - name: Jason Gors  # 60 commits, issues: opened 2 participated in 1
 #   orcid: 
 #   affiliation:  # add full names if not yet listed, or indexes if already are
-# - name: Dave MacFarlane  # 30 commits, issues: opened 4 participated in 4
+- name: Dave MacFarlane  # 30 commits, issues: opened 4 participated in 4
```
Comment (yarikoptic) [inline paper.md]:
```suggestion
 - name: Dave MacFarlane  # 30 commits, issues: opened 4 participated in 4
```

File: paper.md [hunk 2]
```diff
@@ -29,9 +29,9 @@ authors:
...
 #   orcid: 
-#   affiliation:  # add full names if not yet listed, or indexes if already are
+    affiliation: 14 # add full names if not yet listed, or indexes if already are
```
Comment (yarikoptic) [inline paper.md]:
```suggestion
   affiliation: 14 # add full names if not yet listed, or indexes if already are
```

File: paper.md [hunk 3]
```diff
@@ -29,9 +29,9 @@ authors:
 # - name: Jason Gors  # 60 commits, issues: opened 2 participated in 1
 #   orcid: 
 #   affiliation:  # add full names if not yet listed, or indexes if already are
-# - name: Dave MacFarlane  # 30 commits, issues: opened 4 participated in 4
+- name: Dave MacFarlane  # 30 commits, issues: opened 4 participated in 4
 #   orcid:
```
Comment (yarikoptic) [inline paper.md]:
```suggestion
```

**Review by yarikoptic — CHANGES_REQUESTED** [review]
tried to wrangle indentation while in review mode

**Pull request merged**